fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -992,31 +1012,76 @@ private void analyzeBuiltinAggFunction(Analyzer analyzer) throws AnalysisExcepti
             if (ConnectContext.get() != null) {
                 blockEncryptionMode = ConnectContext.get().getSessionVariable().getBlockEncryptionMode();
                 if (fnName.getFunction().equalsIgnoreCase("aes_decrypt")
-                        || fnName.getFunction().equalsIgnoreCase("aes_encrypt")) {
+                        || fnName.getFunction().equalsIgnoreCase("aes_encrypt")
+                        || fnName.getFunction().equalsIgnoreCase("aes_decrypt_v2")
+                        || fnName.getFunction().equalsIgnoreCase("aes_encrypt_v2")) {
```

Comment:
maybe should add some desc to explain old function and v2 function usage in docs

Reply:
done